### ISSUE TITLE
YYYY-DD format (2017-10)

### ISSUE THREAD

**Issue by txthanh82** [xdan/datetimepicker#615]
Would you please help me about display only year and months?
I try:

jQuery.datetimepicker.setLocale('ja');
$(document).find('.datejp2 input').each(function(){
      $(this).datetimepicker({
            format: "Y-m",
            timepicker: false
      });
 });

but datetimepicker is still displayed days of month.
I want to display only year and months.
Please help me.
Thank you very much everyone.

**Comment by swierczek** [comment]
I don't think this has the functionality to display only months in the picker. What the format is doing is changing the value that's input into the field. So if you selected "May 12, 2018" with the format "Y-m" then your final value would just be "2018-05", but the user would still need to select a date in order to input that value.  I think you're looking for https://github.com/KidSysco/jquery-ui-month-picker as a year/monthpicker.
